model_zoo/mnist/mnist_pytorch.py

```diff
@@ -131,15 +107,14 @@ def train(args):
     train_data = torchvision.datasets.ImageFolder(args.training_data)
```

Comment:
Does each worker store a complete dataset locally， and the worker selects the required data from the local dataset through the dataset shard index which is get from data_shard_service?

Reply:
No, each worker can only need to access the dataset storage. For example, we can store the dataset in the remote distribution storage like glusterfs, NAS or HDFS which each worker can access. In the example, I store the dataset on each worker because the dataset is small.